sdk/src/test/java/io/horizen/utxo/api/http/SimpleCustomApi.java

```diff
@@ -34,7 +34,7 @@ public List<Route> getRoutes() {
     private ApiResponse getNSecretsFunction(GetSecretRequest ent) {
         try {
             RespAllSecret resp = new RespAllSecret();
-            List<Secret> res = new ArrayList<>(getFunctionsApplierOnSidechainNodeView().applyFunctionOnSidechainNodeView(v -> v.getNodeWallet().allSecrets()));
+            List<io.horizen.secret.Secret> res = new ArrayList<>(getFunctionsApplierOnSidechainNodeView().applyFunctionOnSidechainNodeView(v -> (v).getNodeWallet().allSecrets()));
```

Comment:
why do we need any changes in this line?

Reply:
No need indeed